kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/KafkaProxyReconciler.java

```diff
@@ -166,11 +165,11 @@ private static InformerEventSource<?, KafkaProxy> buildKafkaServiceInformer(Even
     }
 
     @VisibleForTesting
-    static @NonNull SecondaryToPrimaryMapper<KafkaService> kafkaServiceRefToProxyMapper(EventSourceContext<KafkaProxy> context) {
+    static SecondaryToPrimaryMapper<KafkaService> kafkaServiceRefToProxyMapper(EventSourceContext<KafkaProxy> context) {
         return kafkaServiceRef -> {
             // find all virtual clusters that reference this kafkaServiceRef
 
-            var proxyRefs = ResourcesUtil.resourcesInSameNamespace(context, kafkaServiceRef, VirtualKafkaCluster.class)
+            Set<? extends LocalRef<KafkaProxy>> proxyRefs = ResourcesUtil.resourcesInSameNamespace(context, kafkaServiceRef, VirtualKafkaCluster.class)
```

Comment:
out of curiosity was there a particular reason for going from `var` to `Set<? extends LocalRef<KafkaProxy>>`

Reply:
Yes, the explicit type is less specific that would be inferred for `var`, but it makes a later warning (about dubious .equals on different types) go away. The warning is incorrect because of the equals semantic we have on `LocalRef` applying to subclasses.